### ISSUE TITLE
OSError: [Errno 2] No such file or directory

### ISSUE THREAD

**Issue by noamtcohen** [common-tater/wkwebview-webrtc-shim#3]
I'm trying to build the project with this command:
```
bin/build-libwebrtc
```
getting this error:
```
OSError: [Errno 2] No such file or directory: '/Users/******/wkwebview-webrtc-shim-master/build/src/third_party/_gclient_junit-jar_E6JT6C'
```

looking a bit back I see that this repo  doesn't exist: https://chromium.googlesource.com/external/webrtc/deps/third_party/junit
```fatal: remote error: Git repository not found```

**Comment by jessetane** [comment]
not possible that somehow you are missing depot_tools, right? or maybe try running gclient sync to make sure dependencies are up to date? I don't have time to look at this today but I'll follow up soon.

**Comment by noamtcohen** [comment]
depot_tools are installed but when I run ```gclient sync``` I get: ```Error: client not configured; see 'gclient config'```
I'm not sure how use gclient config in this case.

**Comment by noamtcohen** [comment]
junit repo seems to have been move to here: https://chromium.googlesource.com/external/junit
```git clone https://chromium.googlesource.com/external/junit``` and place the repo in: ```wkwebview-webrtc-shim-master/build/src/third_party``` and the build script continues...
and fails again next time I try with the same error...

**Comment by noamtcohen** [comment]
Maybe you can give me a link to a pre built ```WKWebViewWebRTCShim``` ?

**Comment by jessetane** [comment]
Sorry for the delay getting back to you, I know building chromium can be a pain and I've also run into weird issues where my checkout for whatever reason just can't seem to be updated. Starting again from scratch and doing a fresh checkout generally seems to work though (if you have the time).

**Comment by noamtcohen** [comment]
Thank you for getting back to me, so I'll checkout ```wkwebview-webrtc-shim``` again and run ```bin/build-libwebrtc``` and I'll tell you how it goes (went?).

**Comment by jessetane** [comment]
Oh, I think you just have to blow away the chromium checkout, not the whole
library - should just be a directory called build

**Comment by jessetane** [comment]
You know, it might be that the branch used in the build script is just no
longer compatible with the latest depot_tools. The patches mentioned in the
comments have probably already landed... I don't have time to dig into it
right now but it might be easy to enough to fix, see this bit:
https://github.com/common-tater/wkwebview-webrtc-shim/blob/master/bin/build-libwebrtc#L91-L107

**Comment by noamtcohen** [comment]
Ok, I've tried to checkout again but same issue. I'll take a look at link you've sent me.

**Comment by jessetane** [comment]
Yeah, try building from release 55 without the cherry picking, might work.
